### PR TITLE
feat: add/expose cli flags

### PR DESCRIPTION
**Pull request by hans-m-song**
Every time I execute commands, I would like to opt out of force-updates

This adds additional flags to the cli:

- `--skip-deps` - hoisted up to the global level
- `--disable-force-update` - exposes the ability to disable the force-update flag

Resolves roboll/helmfile/issues/795
Relates to https://github.com/roboll/helmfile/pull/1494#pullrequestreview-506233792

Still a rookie at go (especially a repo of this size), any feedback is appreciated

### CODE REVIEW

**Comment by yxxhero** [comment]
@hans-m-song Thanks for your PR. if it's ready. I will review that and give some feedback.

**Comment by hans-m-song** [comment]
thanks @yxxhero, looking forward to it 😄

File: pkg/config/apply.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func (a *ApplyImpl) SkipCleanup() bool {
 
 // SkipDeps returns the skip deps.
 func (a *ApplyImpl) SkipDeps() bool {
-	return a.ApplyOptions.SkipDeps
+	return a.GlobalOptions.SkipDeps || a.ApplyOptions.SkipDeps
```
Comment (yxxhero) [inline pkg/config/apply.go:149]: remove `SkipDeps` field in `ApplyOptions`. others are similar.

Comment (hans-m-song) [reply pkg/config/apply.go:149]: for my understanding... this is because `ApplyImpl` embeds `GlobalImpl` which embeds `GlobalOptions`?

**Comment by yxxhero** [comment]
@hans-m-song ah. you got me.

**Comment by yxxhero** [comment]
@hans-m-song  you can refer to the logic of `--env`.

**Comment by hans-m-song** [comment]
@yxxhero sorry not sure I follow 🤔 

`--skip-Deps` and `--disable-force-update` are registered in `root` where you have linked to `--environment`

https://github.com/helmfile/helmfile/pull/771/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR122-R123

`DisableForceUpdate` is accessible in `GlobalImpl` similar to how `Environment` is with `Env()`, are you saying I should expose `SkipDeps` in the same way?

https://github.com/helmfile/helmfile/pull/771/files#diff-3ec397d110aac0e46127983382e5ef242ff19068d7390217684559aaeff8a364R139-R142

**Comment by yxxhero** [comment]
@hans-m-song if added in global level. then other subcommands don't need to add.

File: pkg/config/global.go [hunk 2]
```diff
@@ -132,6 +136,11 @@ func (g *GlobalImpl) EnableLiveOutput() bool {
 	return g.GlobalOptions.EnableLiveOutput
 }
 
+// DisableForceUpdate return when to disable forcing updates to repos upon adding
+func (g *GlobalImpl) DisableForceUpdate() bool {
```
Comment (yxxhero) [inline pkg/config/global.go:140]: add SkipDeps function. remote in other subcommands.

Comment (yxxhero) [reply pkg/config/global.go:140]: @hans-m-song

**Comment by yxxhero** [comment]
@hans-m-song final step. please update the docs: https://github.com/helmfile/helmfile/blob/main/docs/index.md#cli-reference

**Review by yxxhero — APPROVED** [review]

**Pull request merged**